src/shared/components/iframe-runtime.tsx

```diff
@@ -101,13 +101,15 @@ export const IframeRuntime: React.FC<IProps> =
         phoneRef.current.disconnect();
       }
     };
-  }, [url, authoredState, report, id, navImageUrl, navImageAltText]);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  },[]);
 
+  const iframeStyle = wrapper === "scaler" ? iframeStyling : {width: "100%", height: iframeHeight, border: "none"};
   return (
-    <div>
-      <iframe ref={iframeRef} src={url} width="100%" height={iframeHeight} frameBorder={0} />
+    <>
+      <iframe ref={iframeRef} src={url} style={iframeStyle} className={wrapper === "scaler" ? css.subInteractiveIframe : ""} />
```

Comment:
Can the `css.subInteractiveIframe` styles just be added to the `iframeStyling` passed by the scaling/fullscreen interactive?  

Then there won't be a need for this duplicate way of setting styles, and the `wrapper` property can be removed.